lib/VMwareWebService/MiqVimVm.rb

```diff
@@ -751,7 +751,7 @@ def addDisk(backingFile, sizeInMB, label=nil, summary=nil)
 							# bck.diskMode = VirtualDiskMode::Independent_nonpersistent
 							bck.diskMode		= VirtualDiskMode::Independent_persistent
 						    bck.split			= "false"
-						    bck.thinProvisioned	= "false"
+						    bck.thinProvisioned	= thinProvisioned.to_s
```

Comment:
I think this will fail miserably if caller passes in anything besides true/false.  Prefer the following to ensure that there are  no bad parameters to API.
`bck.thinProvisioned	= thinProvisioned ? "true" : "false"`